clients/web/src/components/console/log-event-entry.tsx

```diff
@@ -0,0 +1,75 @@
+import { Show, untrack } from "solid-js";
+import { getLogMetadata } from "~/lib/console/get-log-metadata";
+import { timestampToDate, formatTimestamp } from "~/lib/formatters";
+import { getLevelClasses } from "~/lib/console/get-level-classes";
+import { useMonitor } from "~/context/monitor-provider";
+import { processFieldValue } from "~/lib/span/process-field-value";
+import type { LogEvent } from "~/lib/proto/logs";
+import clsx from "clsx";
```

Comment:
`clsx` is actually unnecessary.
Solid's JSX has a `classList` attribute that works the exact same way.